css/style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: "oppins", sans-serif;
+  background-image: url("../img/background.png");
+  background-repeat: no-repeat;
+}
+
+.container {
+  margin: 5%;
+  padding-top: 3rem;
+}
```

Comment:
- [ ] Based on the requirement, kindly use CSS `flexbox` to align the children's contents of the headline section. Using flexbox makes your design responsive across different screen sizes. 👍

Reply:
done :heavy_check_mark: